config/seatunnel.yaml

```diff
@@ -22,6 +22,7 @@ seatunnel:
     queue-type: blockingqueue
     print-execution-info-interval: 60
     print-job-metrics-info-interval: 60
+    log-privacy-keyword: username,password,passwd,token,jwt,secret
```

Comment:
At this stage, there is no need to introduce this parameter, because the configuration printing config is effective for all engines, but this configuration file is only used for zeta.